xrdp/xrdp.ini.in

```diff
@@ -229,12 +229,22 @@ password=ask
 #delay_ms=2000
 
 [neutrinordp-any]
-name=neutrinordp-any
+name=RDP Proxy (NeutrinoRDP)
+; To use this section, you should build xrdp with configure option
+; --enable-neutrinordp.
 lib=libxrdpneutrinordp.@lib_extension@
 ip=ask
 port=ask3389
 username=ask
 password=ask
+; You can Uncomment the following line to enable PAM authentication for Proxy 
+; connections.You may want to change the <ls_btn_ok_y_pos> and <ls_btn_cancel
+; _y_pos> parameters to around 400 in the [Globals] section because the dialog
+; box will collapse.
+#pamusername=ask
+#pampassword=ask
+#pamsessionmng=127.0.0.1
+#delay_ms=2000
```

Comment:
Remove the `delay_ms` line from here as it only applies to VNC connections (see b66aafcd31b086b8d5eb1fe73e083d0b24ddb5fc)

Reply:
Fixed in #1f09ad9.